lib/apiservers/engine/backends/container.go

```diff
@@ -668,6 +696,10 @@ func (c *Container) dockerContainerCreateParamsToPortlayer(cc types.ContainerCre
 	config.Image = new(string)
 	*config.Image = layerID
 
+	// Repo Requested
+	config.RepoName = new(string)
```

Comment:
In this line - and in line 696 - I don't understand why we're setting config.Image to a pointer to an empty string and then resetting it to a different string pointer in the line below. Isn't the empty string assignment unnecessary? Or am I missing something?